packages/create-sitecore-jss/src/templates/nextjs-personalize/src/components/CdpIntegrationScript.tsx

```diff
@@ -20,34 +20,25 @@ function createPageView(locale: string | undefined, routeName: string) {
       pos: pos,
     };
 
-    // eslint-disable-next-line @typescript-eslint/no-empty-function
     Boxever.eventCreate(pageViewEvent, function () {}, 'json');
   });
 }
 
-interface CdpIntegrationProps {
-  pageEditing: boolean | undefined;
-  route: RouteData;
-}
-
-const CdpIntegrationScript = ({
-  route: { itemLanguage, name },
-  pageEditing,
-}: CdpIntegrationProps): JSX.Element => {
-  const clientKey = process.env.BOXEVER_CLIENT_KEY
-  const targetUrl = process.env.BOXEVER_TARGET_URL
+const CdpIntegrationScript = (): JSX.Element => {
 
+  const { sitecoreContext } = useSitecoreContext();
+  const isEditing = sitecoreContext && sitecoreContext.pageEditing;
```

Comment:
Do we need to check for `sitecoreContext`? For example on the line 37 you don't check for `sitecoreContext`